File: lib/src/model/get_bucket_info_result.dart
import 'oss_bucket_summary.dart';
import 'oss_result.dart';

class GetBucketInfoResult extends OSSResult {
  OSSBucketSummary? bucket;

  @override
  String toString() {
    return "GetBucketInfoResult<${super.toString()}>:\n bucket:${bucket?.toString()}";
  }
}

File: lib/src/model/GetBucketInfoResult.java
package com.alibaba.sdk.android.oss.model;

 class GetBucketInfoResult extends OSSResult {
     OSSBucketSummary bucket;

     OSSBucketSummary getBucket() {
        return this.bucket;
    }

     void setBucket(OSSBucketSummary bucket) {
        this.bucket = bucket;
    }

    @override
     String toString() {
        return String.format("GetBucketInfoResult<%s>:\n bucket:%s", super.toString(), bucket.toString());
    }
}
